FEATURES.md

```diff
@@ -182,9 +182,11 @@ Legend:
 |Fullscreen                       |✔️     |✔️     |✔️         |✔️             |**N/A**|✔️     |✔️        |
 |Fullscreen toggle                |✔️     |✔️     |✔️         |✔️             |**N/A**|✔️     |✔️        |
 |Exclusive fullscreen             |✔️     |✔️     |✔️         |**N/A**         |❌    |✔️     |**N/A**|
-|HiDPI support                    |✔️     |✔️     |✔️         |✔️             |▢[#721]|✔️    |**N/A**|
+|HiDPI support                    |✔️     |✔️     |✔️         |✔️             |▢[#721]|✔️    |▢ ^1|
```

Comment:
I added a note below the table:

>     ^1: `WindowEvent::ScaleFactorChanged` is not sent on `stdweb` backend.

 but I suspect it might not be very noticeable.

Reply:
Hm, maybe using a check with a `*1` would get the point across better?

Reply:
I've changed it a bit. Here is how it looks now: https://github.com/rust-windowing/winit/blob/d961343a0f49180953109c8645afbb98645e6092/FEATURES.md#windowing-1